app/controllers/api/v1/rooms_controller.rb

```diff
@@ -5,6 +5,7 @@ module V1
     class RoomsController < ApplicationController
       skip_before_action :verify_authenticity_token # TODO: amir - Revisit this.
       before_action :find_room, only: %i[show start]
+      before_action :ensure_authenticated, only: :create
```

Comment:
Don't forget to remove this

Reply:
Do you want me to call `#ensure_authenticated` manually in the `#create`?

Reply:
That will make it:
`
def create
  return ensure_authenticated
  # Rest of the code...
end
`

Reply:
Nono let's drop the ensure authenticated completely from this. 

None of our other actions are secured yet, so it's better if we do them one shot

Reply:
Alright, I'll remove the check entirely.